app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -127,6 +127,11 @@ def textual_group_miq_custom_attributes
   def textual_miq_custom_attributes
```

Comment:
@enoodle I don't think this should have been named `textual_miq_custom_attributes` in the first place. It should reflect what we're doing here, which is displaying the additional metadata (I don't recall if with @zgalor we had a decision on how to call this). Then, of course the filtering below (only `metadata` section) would be much more understandable. Also, no need to have dependency on models with `displayed_custom_attribute_sections`.